.github/workflows/rosa-scaling-benchmark.yml

```diff
@@ -94,6 +94,7 @@ env:
   PROJECT: runner-keycloak
   ANSIBLE_CUSTOM_VARS_ARG: '-e @env_rosa_benchmark.yml'
   CLIENTS_PER_REALM: 20000
+  GATLING_CLIENTS_PER_REALM: 10000
```

Comment:
Why are we distinguishing between clients created and clients used in load running? Should we just change the variable CLIENTS_PER_REALM to 10k?

Reply:
I though we still wanted to create 20k but only use 10k. If I'm wrong, then I can remove this variable.

Reply:
I don't think there is much difference between creating 20k and 10k so I would just use 10k everywhere to save time creating the additional 10k. Also, we previously used 1 client-per-realm everywhere, so 10k is still better.